MapleServer2/Enums/ItemRarity.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MapleServer2.Enums
+{
+    public enum RarityType
```

Comment:
is this used anywhere? or is it just placeholder? If this is used, are these the same numbers nexon uses?

Reply:
These aren't being used atm. But will be later on. and yes these are the same numbers nexon uses.